Source/monster.h

```diff
@@ -210,13 +210,13 @@ struct MonsterStruct { // note: missing field _mAFNum
 	bool IsWalking() const;
 };
 
-extern int monstkills[MAXMONSTERS];
-extern int monstactive[MAXMONSTERS];
-extern int nummonsters;
+extern int MonsterKills[MAXMONSTERS];

```

Comment:
I can reshuffle them a bit, but moving them across files is probably going to happen in a later PR as that would change what headers are needed in various files and that feels like it could balloon a bit.

Reply:
`MonsterKillCounts`